tests/system/common.py

```diff
@@ -819,3 +819,11 @@ def kill_process_on_host(hostname, pattern):
     else:
         print("Killed no pids")
     return pids
+
+
+# since exhibitor might take ~ 20 minutes to start up, we need to wait for it
+@retrying.retry(wait_fixed=1000, stop_max_attempt_number=1200, retry_on_exception=ignore_exception)
```

Comment:
do we really want to wait 20 minutes for exhibitor? What kind of software starts up 20 minutes? That's crazy...

Shouldn't we maybe give up earlier but fail with message that would clearly state that we failed waiting on exhibitor?

Btw. the usage of exhibitor in shakedown is kind of questionable too. Waiting 20 minutes for current number of masters... can't we get that information from somewhere else?